packages/web3/test/integration/setup.js

```diff
@@ -19,6 +19,6 @@ along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 // eslint-disable-next-line @typescript-eslint/no-require-imports
 require('../config/setup');
 
-const jestTimeout = 15000;
+const jestTimeout = 30000;
```

Comment:
Do you have a reason to increase the timeout?

Reply:
Because there are new integration tests added to the package. And so the tests now take longer.

Reply:
I just updated the test where I kept the updated time (30 second), but I skipped the `in3` test if it took more than `25` seconds. And in that case, I logged a warning to the console.